packages/tooling/adaptive-ui-explorer/src/app.ts

```diff
@@ -0,0 +1,358 @@
+import {
+    attr,
+    css,
+    customElement,
+    FASTElement,
+    html,
+    observable,
+    ref,
+    repeat,
+    ViewTemplate,
+} from "@microsoft/fast-element";
+import { DesignToken } from "@microsoft/fast-foundation";
+import {
+    accentBaseColor,
+    accentPalette,
+    LayerBaseLuminance,
+    layerFillBaseLuminance,
+    layerFillFixedBase,
+    layerFillFixedMinus1,
+    layerFillFixedMinus2,
+    layerFillFixedMinus3,
+    layerFillFixedMinus4,
+    layerFillFixedPlus1,
+    neutralBaseColor,
+    neutralPalette,
+    Palette,
+    Swatch,
+    SwatchRGB,
+} from "@microsoft/adaptive-ui";
+import { ComponentType } from "./component-type.js";
+import { ColorBlock } from "./components/color-block.js";
```

Comment:
You should be able to use the following pattern and then omit the component statements:

```ts
import "./components/color-block.js"; // change to this

// ColorBlock; remove this
```